src/Kerbalism.csproj

```diff
@@ -28,7 +29,7 @@
     <FileAlignment>512</FileAlignment>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <OutputPath>C:\Games\KSP\GameData\Kerbalism\</OutputPath>
+    <OutputPath Condition="Exists('$(KSPDEVDIR)\GameData\Kerbalism\')">$(KSPDEVDIR)\GameData\Kerbalism\</OutputPath>
```

Comment:
Can you add an OutputPath which occurs if it DOES not exist and just dumps it into the Kerbalism folder

Reply:
Sorted